### PR TITLE
Disable keep-padding option in shfmt

### PR DESCRIPTION
**Pull request by NoahGorny**
Has some serious bugs and not really needed

## Description

Disable the -kp option in shfmt by disabling it in our editorconfig
Also fixed all files that need to be changed after this

## Motivation and Context

This was needed as -kp is buggy (https://github.com/mvdan/sh/issues/652)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] If I have added a new file, I also added it to ``clean_files.txt`` and formatted it using ``lint_clean_files.sh``.
- [ ] I have added tests to cover my changes, and all the new and existing tests pass.

### CODE REVIEW

File: themes/base.theme.bash [hunk 1]
```diff
@@ -207,8 +207,10 @@ function git_prompt_minimal_info {
 }
 
 function git_prompt_vars {
-	if ${SCM_GIT_USE_GITSTATUS} && _command_exists gitstatus_query && gitstatus_query && [[ "${VCS_STATUS_RESULT}" == "ok-sync" ]]; then # use faster gitstatus
-		SCM_GIT_GITSTATUS_RAN=true # use this in githelpers and below to choose gitstatus output
+	if ${SCM_GIT_USE_GITSTATUS} && _command_exists gitstatus_query && gitstatus_query && [[ "${VCS_STATUS_RESULT}" == "ok-sync" ]]; then
+		# we can use faster gitstatus
+		# use this variable in githelpers and below to choose gitstatus output
+		SCM_GIT_GITSTATUS_RAN=true
```
Comment (davidpfarrell) [inline themes/base.theme.bash:213]: Was this change related to the keep-padding setting?

Comment (NoahGorny) [reply themes/base.theme.bash:213]: Yes, as `shfmt` did a pretty wierd formatting to the comment, so I preferred to change it entirely

**Comment by NoahGorny** [comment]
@cornfeedhobo @nwinkler @davidpfarrell bump

**Review by davidpfarrell — APPROVED** [review]
Overall I think the new formatting changes on the affected files are better now than before, and doesn't seem to be a big change in general, so I'm good with it.

**Comment by cornfeedhobo** [comment]
Same. And it sounds like mvdan doesn't want to support this feature anymore anyways.

**Review by cornfeedhobo — APPROVED** [review]

**Pull request merged**